views/profile_barista.ejs

```diff
@@ -25,6 +25,32 @@
       <% }) %>
     <% } %>
   </div>
+  <label for="photo" class="label">My Photo</label>
+  <div class="max-w-sm py-4" id="photoView">
+    <% if(barista.photo) { %>
+    <%- barista.photo %>
+    <% } %>
+  </div>
+  <div class="max-w-sm py-4" id="photoUploadResult" hidden></div>
+  <div class="collapse" >
+    <input type="checkbox" onclick="fetchSignature()"/> 
+    <div class="collapse-title btn btn-secondary text-xl font-medium capitalize" >
+      Click to upload your photo

```

Comment:
Related to my note about `fetchSignature` and `uploadProfilePhoto` above: you've tightly coupled your UI to the structure of the functions in `main.js`. What if the stakeholders decided they didn't want this in a collapse? You'd have to rewrite the functions that call the api. IMO, the `collapse` should be a purely visual aspect of the UI, and form itself should contain all the logic needed to call the api.